app/commands/v2/put_content.rb

```diff
@@ -1,6 +1,7 @@
 module Commands
   module V2
     class PutContent < BaseCommand
+      ITEM_NOT_FOUND = Class.new
```

Comment:
It's a sentinel object; the explanation is in the commit message but maybe it would be clearer if there was a comment here as well.